astropy/visualization/wcsaxes/axislabels.py

```diff
@@ -59,7 +60,7 @@ def set_visibility_rule(self, value):
     def get_visibility_rule(self):
         return self._visibility_rule
 
-    def draw(self, renderer, bboxes, ticklabels_bbox,
```

Comment:
Is this API change? I'll let the subpackage maintainers decide.

If it is, this needs a change log, and possibly some sort of deprecation?

Reply:
Yes, possibly. I can always put the function signature back to what it was, and issue a deprecation warning that the parameter I've changed doesn't do anything.

Reply:
Not sure if there's another canonical way to deprecate a parameter, but I couldn't find anything in the docs so I've brewed my own solution. Let me know if there's a better way!

Reply:
We have https://docs.astropy.org/en/latest/api/astropy.utils.decorators.deprecated_renamed_argument.html but @larrybradley found bugs with it before, so not sure if I should recommend it or not.